README.md

```diff
@@ -40,28 +40,28 @@ See [source](examples/example-shader/src/lib.rs) for full details.
 
 ## Background
 
-Historically in games GPU programming has been done through writing either HLSL, or to a lesser extent GLSL. These are simple programming languages that have evolved along with rendering APIs over the years. However, as game engines have evolved, these languages have failed to provide mechanisms for dealing with large codebases, and have generally stayed behind the curve compared to other programming languages.
+Historically in games, GPU programming has been done through writing either HLSL or to a lesser extent GLSL. These are simple programming languages that have evolved along with rendering APIs over the years. However, as game engines have evolved, these languages have failed to provide mechanisms for dealing with large codebases, and have generally stayed behind the curve compared to other programming languages.
```

Comment:
this comma is incorrect; there should either be no comma like the original or there should also be a comma after Historically.